epictrack-api/migrations/versions/d8b405f9d663_create_pip_org_type_table.py

```diff
@@ -0,0 +1,54 @@
+"""create pip_org_type table
+
+Revision ID: d8b405f9d663
+Revises: 47b41964f6df
+Create Date: 2023-10-18 11:45:36.531064
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'd8b405f9d663'
+down_revision = '47b41964f6df'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('pip_org_types',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('created_by', sa.String(length=255), nullable=True),
+    sa.Column('created_at', sa.DateTime(timezone=True), server_default=sa.text("TIMEZONE('utc', CURRENT_TIMESTAMP)"), nullable=True),
+    sa.Column('updated_by', sa.String(length=255), nullable=True),
```

Comment:
It would be good to have the data to put in with the migration script. Cz this is master data, this has to be copied to all different environments .